utils.py

```diff
@@ -0,0 +1,21 @@
+def get_progress_emoji(ratio):
```

Comment:
Can we us a size 11 list?
emoji = [':new moon', ':new moon', ':waning_crescent_moon:', ':waning_crescent_moon:', ...,':full_moon:']

Then, you can simply return emoji [(int)ration/10]. What do you think?